invenio_audit_logs/services/generators.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio-Audit-Logs Permissions Generators."""
+
+from flask import g
+from flask_principal import UserNeed
+from invenio_records_permissions.generators import Generator
+
+
+# Permission generator to check if the user identity matches the identity in the request
+class CurrentUser(Generator):
```

Comment:
why do we need to double check? isn't AnyUser enough ?

Reply:
This check is for making sure that the service method can't be called with a different user in the data.